### PR TITLE
document correction

### PR DESCRIPTION
**Pull request by loulin**
There are two activities.allow examples in config.activities, one
should be activities.deny

### CODE REVIEW

File: gruntfile.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports = function(grunt){
         extensions: "js",
         specNameMatcher: "[Ss][Pp][Ee][Cc][Ss]",
         jUnit: {
-          report: true,
+          report: false,
```
Comment (lahvey) [inline gruntfile.js]: I have also got this problem jasmine-contrib/grunt-jasmine-node#52 and I found you already commented there. So is this a temporary solution?

Comment (loulin) [inline gruntfile.js]: I have also got this problem jasmine-contrib/grunt-jasmine-node#52 and I found you already commented there. So is this a temporary solution?

File: specs/helpers/index.js [hunk 2]
```diff
@@ -37,11 +37,11 @@ var helpers = {
   },
 
   notAuthenticated: function(req, res){
-    res.send(403, {});
+    res.status(403).send({});
```
Comment (loulin) [inline specs/helpers/index.js]: fix express deprecated warning

File: mustbe/verifier/index.js [hunk 3]
```diff
@@ -72,6 +72,11 @@ Verifier.prototype.isAuthorized = function(activity, requestParams, cb){
 
     var validator = validators[activity];
     if (!validator){
+      // global "deny" or "allow" defined but not passed
+      if(denyAllowConfig.denier || denyAllowConfig.allower) {
+        return cb(null, false);
+      }
+
```
Comment (loulin) [inline mustbe/verifier/index.js]: To follow the original design, we have to define validators for all activities, except that the global "deny" or "allow" always returns true. But how about plenty of activities which have the same logic? 
For example, we save all the activities in database and assign roles for special users. We can define a global "allow" to check the user role and activity from database. For others who don't pass the "allow", if there has an individual activity, we can continue the check, if not, just go to notAuthorized handle. 
I think there may be a better solution for my concern. Could anyone help to review the changes?

File: specs/custom-identity/authorization-override-specs.js [hunk 4]
```diff
@@ -128,6 +128,49 @@ describe("custom identity authorization overrides", function(){
...
+        response = res;
+        done();
+      });
+    });
+
+    it("should not allow the request", function(){
+      helpers.expectResponseCode(response, 403);
+    });
+  });
+
```
Comment (loulin) [inline specs/custom-identity/authorization-override-specs.js]: test for the added authorization check.

**Comment by derickbailey** [comment]
hi @loulin - these changes are looking good, and i'd like to discuss the authorization check change further ... but there are a lot of unrelated things going on in here. 

can you split these in to separate branches and separate pull requests?  this could be 2 branches / pull requests, which would help to make it easier to discuss and keep things moving forward

0. the first would be the docs update, gruntfile change, demo fix and specs/helpers fix. i don't mind all of those being in one pull request, as they are ancillary to the real code and functionality. 
0. the second pull request should contain the specs change and modification to the verifier/index.js file - i like the direction you are heading with this change. i just want to see it on a separate pull request so it can be reviewed and discussed properly

thanks!

  - Derick

**Comment by derickbailey** [comment]
thanks @loulin! I got these changes pulled in :)

**Pull request merged**